db/migrate/20220719113627_add_tables.rb

```diff
@@ -0,0 +1,19 @@
+class AddTables < ActiveRecord::Migration[7.0]
+  def change
+    create_table :task_lists do |t|
+      t.boolean :hidden, default: false
+
+      t.timestamps
+    end
+
+    create_table :tasks do |t|
+      t.string :name, null: false
+      t.string :description
+      t.timestamp :deadline, null: false
```

Comment:
Do you really want to `null: false`. Some tasks don't need deadline, deadline should be optional IMHO